### ISSUE TITLE
[Problem] Link for the "WebTools" in the "Addon-Manager" does not match.

### ISSUE THREAD

**Issue by poulhein** [FreeCAD/FreeCAD#11252]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

The link in the Addon-Manager shows "https://github.com/yorikvanhavre/WebTools/blob/main/README.md"
Afaik it should be "https://github.com/yorikvanhavre/WebTools/blob/master/README.md"

### Full version info

```shell
[code]
OS: Windows 10 build 19045
Word size of FreeCAD: 64-bit
Version: 0.21.1.33668 +26 (Git)
Build type: Release
Branch: (HEAD detached at 0.21.1)
Hash: f6708547a9bb3f71a4aaade12109f511a72c207c
Python 3.8.10, Qt 5.15.2, Coin 4.0.1, Vtk 8.2.0, OCC 7.6.3
Locale: German/Germany (de_DE)
Installed mods: 
  * BIM 2021.12.0
  * dodo 1.0.0
  * fasteners 0.4.68
  * parts_library
  * Reinforcement
  * Reporting
[/code]
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
Yes, thanks for the heads up. Note:  the ticket is addressing a 3rd party addon and needs to be  opened in 
https://github.com/yorikvanhavre/WebTools

CC @yorikvanhavre problem is in https://github.com/yorikvanhavre/WebTools/commit/8a288f90f726d56fd68f4133b4a303ea023bcf6a 
I'll make a PR fix
